include/miopen/miopen.h

```diff
@@ -381,6 +381,15 @@ typedef enum {
     miopenPoolingAverageInclusive = 2, /*!< Inclusive Average pooling */
 } miopenPoolingMode_t;
 
+/*! @ingroup pooling
+ * @enum miopenPoolingWorkspaceIndexMode_t
+ * Pooling layer workspace index mode.
```

Comment:
Can we have more description here? Or in the API usage.

Reply:
You can use the string `[skip ci]` if you add comments to prevent the CI from running again.

Reply:
Done